metpy/io/_tools.py

```diff
@@ -14,14 +12,7 @@
 
 log = logging.getLogger(__name__)
 
-
-# This works around problems on early Python 2.7 where Struct.unpack_from() can't handle
-# being given a bytearray; use memoryview on Python 3, since calling bytearray again isn't
-# cheap.
-try:
-    bytearray_to_buff = buffer
-except NameError:
-    bytearray_to_buff = memoryview
+bytearray_to_buff = memoryview
```

Comment:
Can we just replace all calls to `bytearray_to_buff` with `memoryview` instead?

Reply:
Yup that works